src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -354,51 +323,47 @@ private async Task UpdateAdvertisingManifestAsync(WorkloadManifestInfo manifest,
             }
         }
 
-        private (ManifestVersion ManifestVersion, SdkFeatureBand ManifestFeatureBand, Dictionary<WorkloadId, WorkloadDefinition> Workloads)?
-            GetAdvertisingManifestVersionAndWorkloads(ManifestId manifestId)
+        private (ManifestVersionWithBand ManifestWithBand, WorkloadCollection Workloads)? GetAdvertisingManifestVersionAndWorkloads(ManifestId manifestId)
         {
-            var manifestPath = Path.Combine(GetAdvertisingManifestPath(_sdkFeatureBand, manifestId),
-                "WorkloadManifest.json");
+            var manifestPath = Path.Combine(GetAdvertisingManifestPath(_sdkFeatureBand, manifestId), "WorkloadManifest.json");
             if (!File.Exists(manifestPath))
             {
                 return null;
             }
 
-            using (FileStream fsSource = new(manifestPath, FileMode.Open, FileAccess.Read))
-            {
```

Comment:
It provides no functional purpose as the `using` variable calls `Dispose` at the end of the method in both situations. Both end up with the same call stack. If it needed more encapsulation, another method/function would help, but it doesn't add anything since this logic is the entire point of the method itself.

Reply:
That is fair, I tend to lean towards using {} because of this infamous Apple bug, https://medium.com/swlh/apples-most-notorious-code-bug-6478ebaea44f but it wouldn't change the behavior here, so im cool with this 😄